src/Analysis/Engine/Test/LanguageServerTests.cs

```diff
@@ -433,14 +433,30 @@ public class LanguageServerTests {
 
         [TestMethod, Priority(0)]
         public async Task CompletionForOverrideArgs() {
+            var code = @"
+class A(object):
+    def foo(self, a, b=None, *args, **kwargs):
+        pass
+
+class B(A):
+    def f";
+
             using (var s = await CreateServer()) {
-                var u = await AddModule(s, "class A:\n    def bar(arg=None): pass\n\nclass B(A):\n    def b");
+                var u = await AddModule(s, code);
```

Comment:
To do what actually product does, you can do

```cs
    var u = await server.OpenDefaultDocumentAndGetUriAsync(code);
```

Reply:
Done, though if that's the new "correct" way to do it, then it should probably also be replaced everywhere else.